File: samples-vb/getting-started/Regression_TaxiFarePrediction/TaxiTrip.vb
Imports Microsoft.ML.Runtime.Api

Public Class TaxiTrip
    <Column("0")>
    Public VendorId As String
    <Column("1")>
    Public RateCode As String
    <Column("2")>
    Public PassengerCount As Single
    <Column("3")>
    Public TripTime As Single
    <Column("4")>
    Public TripDistance As Single
    <Column("5")>
    Public PaymentType As String
    <Column("6")>
    Public FareAmount As Single
End Class

File: samples/csharp/getting-started/Regression_TaxiFarePrediction/TaxiFarePrediction/TaxiFarePredictionConsoleApp/DataStructures/TaxiTrip.cs

using Microsoft.ML.Data;

namespace Regression_TaxiFarePrediction.DataStructures
{
    public class TaxiTrip
    {
        [LoadColumn(0)]
        public string VendorId;

        [LoadColumn(1)]
        public string RateCode;

        [LoadColumn(2)]
        public float PassengerCount;

        [LoadColumn(3)]
        public float TripTime;

        [LoadColumn(4)]
        public float TripDistance;

        [LoadColumn(5)]
        public string PaymentType;

        [LoadColumn(6)]
        public float FareAmount;
    }
}